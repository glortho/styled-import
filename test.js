const pluginTester = require('babel-plugin-tester')
const plugin = require('babel-plugin-macros')

pluginTester({
  plugin,
  fixtures: __dirname + '/__fixtures__',
})
