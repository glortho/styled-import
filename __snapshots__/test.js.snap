// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`macros removes when not used: removes when not used 1`] = `
"
const styledImport = require('./macro')

      ↓ ↓ ↓ ↓ ↓ ↓


"
`;
